src/test/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMockTest.java

```diff
@@ -112,6 +116,51 @@ public void runTaskTimer_ZeroDelay_DoesntExecuteTaskImmediately()
 		assertEquals(1, count.get());
 	}
 
+	@Test
+	public void runScheduledTask_WhileScheduler_ShutDown()
+	{
+		AtomicInteger count = new AtomicInteger(0);
+		Runnable callback = () -> count.incrementAndGet();
+		scheduler.shutdown();
+		scheduler.runTaskTimer(null, callback, 0, 2L);
+		scheduler.performTicks(1L);
```

Comment:
Here you are shutting down the scheduler and then running your tasks. However, `shutdown` is normally only called by `MockBukkit.unmock()`. So I'm not entirely certain what the point is of this test.

Reply:
proving task dont run

Reply:
I will look later today